src/language-js/comments.js

```diff
@@ -361,8 +362,7 @@ function handleObjectPropertyAssignment({
 }) {
   if (
     enclosingNode &&
```

Comment:
Do you think we can move this to the function? `isCallExpression` and `isMemberExpression` checked in side.

Reply:
Sorry what does "this" mean?

Reply:
```js
function isObjectProperty(node) {
  return node && (node.type === "ObjectProperty" || node.type === "Property");
}
```

So we don't need check `enclosingNode &&` here.

Reply:
Ah I got it.